content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -46,6 +46,9 @@ Data privacy
 |          | [Npgsql][16]                   | {{< X >}}   |             |
 |          | [MySql.Data][17]               |             | {{< X >}}   |
 |          | [MySqlConnector][18]           |             | {{< X >}}   |
+| **PHP**  [dd-trace-php][19] >= 0.86.0 ||             |             |
+|          | [pdo][20]                       | {{< X >}}  | {{< X >}}   |
```

Comment:
Should probably also advertise mssql? Or is mssql not available yet from the DBM side itself?